solr/core/src/java/org/apache/solr/metrics/prometheus/core/SolrCoreSearcherMetric.java

```diff
@@ -27,7 +27,7 @@
 /** Dropwizard metrics of name SEARCHER.* */
 public class SolrCoreSearcherMetric extends SolrCoreMetric {
   public static final String CORE_SEARCHER_METRICS = "solr_metrics_core_searcher_documents";
-  public static final String CORE_SEARCHER_TIMES = "solr_metrics_core_average_searcher_warmup_time";
+  public static final String CORE_SEARCHER_TIMES = "solr_metrics_core_searcher_warmup_time";
```

Comment:
```suggestion
  public static final String CORE_SEARCHER_TIMES = "solr_metrics_core_searcher_warmup_time_ms";
```
Same as above assuming this is in `ms`